playbooks/geaccirc.yml

```
@@ -1,9 +1,12 @@
 ---
-- hosts: geaccirc_prod
+# by default this playbook runs in the staging environment
+# to run in production, pass '-e runtime_env=prod'
+- name: build the geac app
+  hosts: geaccirc_{{ runtime_env | default('staging') }}
   remote_user: pulsys
   become: true
   vars_files:
-    - ../group_vars/geaccirc/geaccirc_prod.yml
+    - ../group_vars/geaccirc/{{ runtime_env | default('staging') }}.yml
```

We probably don't want to default to staging, since there's no staging environment for this project.
```suggestion
    - ../group_vars/geaccirc/{{ runtime_env }}.yml
```